javascripts/discourse/initializers/init-custom-buttons.js

```diff
@@ -0,0 +1,6 @@
+import { apiInitializer } from "discourse/lib/api";
+import HomepageButtons from "../components/homepage-buttons";
+
+export default apiInitializer("1.8.0", (api) => {
```

Comment:
no need for the version arg

```suggestion
export default apiInitializer((api) => {
```